src/player_activity.h

```diff
@@ -133,6 +137,36 @@ class player_activity : public JsonSerializer, public JsonDeserializer
          * Returns true if the activity is complete.
          */
         bool is_complete() const;
+
+        activity_type type = ACT_NULL; /** type of activity. */
```

Comment:
Why did you move and restyle all these members?
Moving them makes the diff hard to review, and this is a deprecated style of comment.